docs/monitoring.md

```diff
@@ -315,22 +315,24 @@ API path listed as below:
+| /workerInfo                                                 | List worker information of the service. It will list all registered workers 's information.                                                                                                        |
+| /lostWorkers                                                | List all lost workers of the master.                                                                                                                                                               |
+| /excludedWorkers                                            | List all excluded workers of the master.                                                                                                                                                           |
+| /shutdownWorkers                                            | List all shutdown workers of the master.                                                                                                                                                           |
+| /threadDump                                                 | List the current thread dump of the master.                                                                                                                                                        |
+| /hostnames                                                  | List all running application's LifecycleManager's hostnames of the cluster.                                                                                                                        |
+| /applications                                               | List all running application's ids of the cluster.                                                                                                                                                 |
+| /shuffles                                                   | List all running shuffle keys of the service. It will return all running shuffle's key of the cluster.                                                                                             |
+| /listTopDiskUsedApps                                        | List the top disk usage application ids. It will return the top disk usage application ids for the cluster.                                                                                        |
+| /exclude?add=${ADD_WORKERS}&remove=${REMOVE_WORKERS}        | Excluded workers of the master add or remove the worker manually given worker id. The parameter add or remove specifies the excluded workers to add or remove, which value is separated by commas. |
+| /sendWorkerEvent?type=${WorkerEventType}&workers=${WORKERS} | For Master(Leader) can send worker event to manager workers.                                                                                                                                       |
```

Comment:
Should clear that work is separated by commas.

Reply:
Also, we should list all types here or link to a web doc that shows all the commands.